src/node_i18n.cc

```diff
@@ -489,8 +492,11 @@ static void ToUnicode(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(args.Length(), 1);
   CHECK(args[0]->IsString());
   Utf8Value val(env->isolate(), args[0]);
+  // optional arg
+  bool lenient = args[1].As<Boolean>()->Value();
```

Comment:
Can you update the `args.Length()` check above to use 2? Also, you probably want to add a `CHECK(args[1]->IsBoolean());` or do `args[1]->BooleanValue()` instead.